catalog/solr/docs/src/main/resources/IntContents.adoc

```diff
@@ -241,7 +76,7 @@ The Solr Catalog Provider is used in conjunction with an Apache Solr Server dat
 |Scalability
 |* Allows the middle-tier to be scaled by pointing
 * various middle-tier instances to one server facade.
```

Comment:
In the Solr Catalog External Provider Pros and Cons table, should the first two Pro bullet points of the Scalability feature be a single bullet point?